index.html

```diff
@@ -104,10 +106,11 @@ <h3>Basic alerts</h3>
             <p class="code-snippet-heading">Code Snippets</p>
             <div class="code-snippet">
               <iframe
+                class="code-snippet-code"
                 src="https://carbon.now.sh/embed/nlm6Xyfb6HW1khg7oZWK"
                 style="
                   width: 100%;
-                  height: 298px;
+                  height: 260px;
```

Comment:
you can remove Inline styling for iframes & can include them in CSS files. Otherwise, nice work 💯